src/qml/pages/node/NodeRunner.qml

```diff
@@ -9,11 +9,17 @@ import "../../controls"
 import "../../components"
 
 Page {
+    signal settingsClicked
+    id: root
     background: null
     clip: true
-    property alias navRightDetail: navbar.rightDetail
-    header: NavigationBar {
-        id: navbar
+    header: NavigationBar2 {
+        rightItem: NavButton {
```

Comment:
This loses it's right padding

| Master | PR |
| ------ | -- |
| <img width="106" alt="Screenshot 2023-09-08 at 2 39 22 AM" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/25a10e25-9afc-4e73-a057-4c4c65da0c27"> | <img width="124" alt="Screenshot 2023-09-08 at 2 36 08 AM" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/e85da5c7-4248-48be-9048-23df92329c84"> |

Reply:
While also exposing the previous padding was not perfect